lib/chef-dk/command/verify.rb

```diff
@@ -465,22 +465,25 @@ def foo
         end
       end
 
-      if Chef::Platform.windows?
-        # TODO UW-6 - We haven't decided exactly where these binaries will live
-        # so we will write this test when we do this card
-        # add_component "git-windows"
-      else
-        add_component "git" do |c|
-          c.base_dir = "embedded/bin"
-          c.smoke_test do
+      add_component "git" do |c|
+        c.base_dir = "embedded/bin"
+        c.smoke_test do
+	  tmpdir do |cwd|
             sh!("#{embedded_bin("git")} config -l")
-          end
-          c.integration_test do
-            tmpdir do |cwd|
-              sh!("#{embedded_bin("git")} clone git@github.com:chef/ffi-yajl.git", cwd: cwd)
-              sh!("#{embedded_bin("git")} clone https://github.com/chef/chef-provisioning", cwd: cwd)
-            end
-          end
+	    sh!("#{embedded_bin("git")} clone https://github.com/chef/chef-provisioning", cwd: cwd)
+
+	    # # If /usr/bin/git is a symlink, fail the test.
+	    # # Note that this test cannot go last because it does not return a
+	    # # Mixlib::Shellout object in the windows case, which will break the tests.
+	    unless Chef::Platform.windows?
+             failure_str = "/usr/bin/git contains a symlink which might mean we accidentally overwrote system git via chefdk."
+             fail_if_exit_zero("test -L /usr/bin/git", failure_str)
```

Comment:
Not every *nix system installs it here. Perhaps we could do a `which git` to determine where it is? But I would like to look at the environment to make sure we don't have `embedded/bin` early in the path.